### ISSUE TITLE
Attachments are misaligned on 0.3.0

### ISSUE THREAD

**Issue by SUPERCILEX** [hashintel/hash#2019]
### Describe the bug

```
Error: Setup required.
├╴at src/run.rs:101:10
╰╴ABC
   
   DEF
```

It looks like the space after `╰╴` was removed which means there are one too many spaces in the rest of the attachment. Maybe removing the space after each line is a mistake? I kinda like the old one better. Old:

```
Error: Setup required.
├╴ src/run.rs:101:10
╰╴ ABC
   
   DEF
```

### To reproduce

Attach a multi-paragraph string.

**Comment by TimDiekmann** [comment]
Thanks for reporting this @SUPERCILEX, this was an oversight. It will be fixed in #2022.

**Comment by SUPERCILEX** [comment]
Awesome, thanks! If it's possible to get a quick 0.3.1 release that would be perfect.

**Comment by TimDiekmann** [comment]
We have no breaking changes so far, so why not 🚀

**Comment by SUPERCILEX** [comment]
Thank you!

**Comment by TimDiekmann** [comment]
[Fixed in 0.3.1](https://github.com/hashintel/hash/releases/tag/error-stack%400.3.1)
